app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt

```diff
@@ -24,10 +25,18 @@ class MeetingRequestService : FirebaseMessagingService() {
    */
   override fun onMessageReceived(remoteMessage: RemoteMessage) {
```

Comment:
Notice:  The if-else blocks for titles could be replaced with a cleaner structure, such as a mapping table using a when statement : 

when (title) {
    MEETING_REQUEST -> {
        Log.d("MEETING REQUEST", "RECEIVED MEETING REQUEST")
        val meetingRequest = MeetingRequest(message = message, senderUID = senderUid)
        MeetingRequestManager.meetingRequestViewModel?.addToMeetingRequestInbox(meetingRequest)
    }
    MEETING_RESPONSE -> Log.d("MEETING RESPONSE", "RECEIVED MEETING RESPONSE")
    MEETING_CONFIRMATION -> Log.d("MEETING CONFIRMATION", "RECEIVED MEETING CONFIRMATION")
    else -> Log.w("MESSAGE RECEIVED", "Unknown title: $title")
}